integrated_channels/integrated_channel/learner_data.py

```diff
@@ -25,26 +25,30 @@ class BaseLearnerExporter(object):
     Base class for exporting learner completion data to integrated channels.
     """
 
+    GRADE_PASSING = 'Pass'
+    GRADE_FAILING = 'Fail'
+    GRADE_INCOMPLETE = 'In Progress'
+
     @property
     def grade_passing(self):
         """
         Returns the string used for a passing grade.
         """
-        return 'Pass'
+        return self.GRADE_PASSING
```

Comment:
I can't recall @pomegranited 's original intent in introducing these property functions. The reason I didn't change this is as follows: If the grades are really default values, it makes sense for them to be constants owned by the base class that can be referenced elsewhere. These property functions are also performing the default behavior - by returning the constants. They can be overridden by channels that do have constraints on the grades we send back. But I agree that it feels clunky to have both - I think I would rather get rid of these functions and keep the constants if we insist on choosing one.